### DISCUSSION TITLE
Option to use a USB-camera

### DISCUSSION THREAD

**Discussion by nilspupils** [interactionresearchstudio/CameraServer#96] [Ideas]
Not a problem but a possible extension of features: 
There are a couple of camera options available for the RPi, which use the camera connector. Apart from these cameras, a huge variety of other either standard or very specialized cameras are available for USB, among them simple webcams and also highly sensitive low light sensor cameras, for example using Sony STARVIS sensors. To be able to use these USB cameras with Mynaturewatch would significantly expand the application possibilities. Full color lowlight for animal detection would be wonderful!

Perfect solution would be an automatic detection of an attatched USB-camera and the option to choose which camera to use for Mynaturewatch in the web interface. 

Thank you for the consideration of the request and keep up the good work!

**Comment by mikevanis** [comment]
The entire video pipeline is built using picamera, so it's very hard to imagine it working with a usb camera. Would a still-only usb camera be desirable?

**Comment by nilspupils** [comment]
Hi @mikevanis, thank you for taking the time to answer! Stills would be absolutely fine, as long as a couple of them would be taken (see my other feature request). Since picamera is replaced by libcamera now would that be a possible solution (or opencv, as it is already installed anyways)?
Thank you very much!

**Comment by mikevanis** [comment]
Note: Converted to discussion as would need support from more users to be considered as a feature.
